### ISSUE TITLE
[FEATURE REQUEST] fastly objectstore get should return the value directly

### ISSUE THREAD

**Issue by acme** [fastly/cli#771]
**Is your feature request related to a problem? Please describe.**

Currently `fastly objectstore get` returns both the key and the value:

```bash
fastly objectstore get --id $OBJECTSTOREID --key "hello"
Key: hello
Value: "there"
```

**Describe the solution you'd like**

I would like it to return just the value so that it can be scripted in a shell pipeline:

```bash
fastly objectstore get --id $OBJECTSTOREID --key "hello"
there
```

**Describe alternatives you've considered**

**Additional context**

**Comment by fgsch** [comment]
This should be solved once #792 lands.

**Comment by fgsch** [comment]
Fixed in https://github.com/fastly/cli/commit/d4bba2c4e0f8d8271b9fb79f4706cbca6b6c86b1
